public/javascripts/Progress/src/Progress.js

```diff
@@ -51,22 +51,26 @@ function Progress (_, $, userRole) {
         });
     }
 
-    function putUserOrg(e) {
+    function putUserTeam(e, newTeam) {
         var parsedId = $(this).attr('id').split("-"); // the id comes in the form of "from-startOrg-to-endOrg"
-        var startOrg = parsedId[1];
-        var endOrg = parsedId[3];
+        var startTeam = parsedId[1];
+        if(newTeam != null){
```

Comment:
From [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-guide): `if(...)){` -> `if (...) {`. One day I'll get the automated linter working!!